### PR TITLE
Trello  documentation for quota by storage class 

### PR DESCRIPTION
**Pull request by mburke5678**
Based on https://github.com/openshift/openshift-docs/pull/3349

@derekwaynecarr PTAL

### CODE REVIEW

File: admin_guide/quota.adoc [hunk 1]
```diff
@@ -85,6 +81,28 @@ exceed this value.
...
 
+.Storage Resources Managed by Quota
+[cols="3a,8a",options="header"]
+|===
+
+|Resource Name |Description
+
+|`*requests.storage*`
+|The sum of storage requests across all persistent volume claims in any state cannot
+exceed this value. `*storage*` and `*requests.storage*` are the same value and can
```
Comment (derekwaynecarr) [inline admin_guide/quota.adoc]: the only supported value is `requests.storage`  - i am not sure why we stated `storage` as interchangeable.  it never has been.

**Review by derekwaynecarr — CHANGES_REQUESTED** [review]
one update requested, then LGTM

**Comment by mburke5678** [comment]
@qwang1 PTAL

**Comment by mburke5678** [comment]
https://trello.com/c/Sbn4CyZz/386-document-quota-ability-to-scope-pvc-quotas-by-storage-class

**Comment by mburke5678** [comment]
@qwang1  PTAL

**Comment by qwang1** [comment]
@mburke5678 LGTM, thanks.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

File: admin_guide/quota.adoc [hunk 2]
```diff
@@ -85,6 +81,27 @@ exceed this value.
...
+exceed this value.
+
+|`*persistentvolumeclaims*`
+|The total number of persistent volume claims that can exist in the project.
+
+|`*<storage-class-name>.storageclass.storage.k8s.io/requests.storage*`
+|The sum of storage requests across all persistent volume claims in any state that have a matching storage class, cannot exceed this value.
+
+|`*<storage-class-name>.storageclass.storage.k8s.io/persistentvolumeclaims*`
+|The total number of persistent volume claims with a matching storage class that can exist in the project .
```
Comment (ahardin-rh) [inline admin_guide/quota.adoc]: looks like there's an extra space before the period

**Comment by ahardin-rh** [comment]
@mburke5678 Just one comment from me. Also, needs a squash. Otherwise, looks good!

**Comment by mburke5678** [comment]
[rev_history]
|xref:../admin_guide/admin_guide/quota.adoc#admin-guide-quota[Setting Quotas]
|Added section on Storage Resources Managed by Quota.
%

**Pull request merged**